.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+## Description
+
+Add a brief summary of your PR here, between 1 sentence and two paragraphs
+
+## Additions
+
+- Detail API additions here
+- Do not list changes or removals
+
+## Removals
+
+- List API removals here
+
+## Modifications
+
+- List changes to existing structures and functions here
+
+## PR Checklist
+
+By placing an x in the boxes I certify that I have
```

Comment:
With this phrasing, it suggests the following list are part of the full sentence. Either need to change this line so the checkboxes are not grammatically dependent on this, or change the checkboxes so they match the grammar flow. The former would be easier, but if possible, the latter would be more professional.